Makefile

```diff
@@ -294,6 +294,7 @@ endif
 
 # Global settings for Windows targets
 ifeq ($(TARGETSYSTEM),WINDOWS)
+  BACKTRACE = 0
```

Comment:
I'd add a blurb for BACKTRACE's usage up top along with the others like TILES, LOCALIZE, etc.

Also, this doesn't catch cygwin, does it? I assume it should given the code has `#if !(defined _WIN32 || defined WINDOWS || defined __CYGWIN__)`.